libs/features/mas/commerce/src/checkout-link.js

```diff
@@ -10,6 +10,12 @@ import { selectOffers, useService } from './utilities.js';
 export const CLASS_NAME_DOWNLOAD = 'download';
 export const CLASS_NAME_UPGRADE = 'upgrade';
 
+/**
+ * HTML Checkout Link Anchor element, extends anchor
+ * 
+ * @element a[data-wcs-osi]
+ * @fires merch:click
```

Comment:
it doesn't fire `merch:click` though.

Reply:
right, this was a test, i'll remove it

Reply:
(click doc will come with click event pr)